### PR TITLE
Ui debug

### PR DESCRIPTION
**Pull request by eoghanTadhg**
Assorted user interface fixes including general and Work section debugging.

### CODE REVIEW

File: packages/thriver-core/lib/client/scroll.js [hunk 1]
```diff
@@ -94,7 +94,8 @@ handleHeaderStateChange = function (event) {
         if ($(document).scrollTop() > 1000 && $(window).width() > 767)
             document.getElementById("back-to-top").classList.add('active');
         else
-            document.getElementById("back-to-top").classList.remove('active');
+            if(document.getElementById("back-to-top"))
```
Comment (micchickenburger) [inline packages/thriver-core/lib/client/scroll.js:97]: Why is this condition needed?

File: packages/thriver-navigation/lib/templates/utility.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
...
     <li class="{{#if equals position 'right'}}right{{/if}} {{#if more}}more{{/if}}">
-        <a href="{{#if equals type 'link'}}{{{url}}}{{/if}}{{#if equals type 'sidebar'}}/{{{sidebar}}}{{/if}}{{#if equals type 'null'}}/{{/if}}" target="{{#if equals type 'external'}}_blank{{/if}}" class="{{#if icon}}icon icon-{{{icon}}}{{/if}} {{#if iconAfter}}icon-after{{/if}} {{#if this.alerts}}{{#if show}}notificationAlert{{/if}}{{/if}}" aria-controls="{{#if equals type 'sidebar'}}{{{sidebar}}}{{/if}}" aria-expanded="false" data-toggle="{{#if equals type 'sidebar'}}canvas{{/if}}" data-action="{{#if action}}{{action}}{{/if}}">
+        <a href="{{#if equals type 'link'}}{{{url}}}{{/if}}{{#if equals type 'sidebar'}}/{{{sidebar}}}{{/if}}{{#if equals type 'null'}}/{{/if}}" target="{{#if equals type 'external'}}_blank{{/if}}" class="{{#if icon}}icon icon-{{{icon}}}{{/if}} {{#if iconAfter}}icon-after{{/if}} {{#if this.alerts}}{{#if show}}notificationAlert{{/if}}{{/if}}" aria-controls="{{#if equals type 'sidebar'}}{{{sidebar}}}{{/if}}" aria-expanded="false" data-toggle="{{#if equals type 'sidebar'}}canvas{{/if}}" data-action="{{#if action}}{{action}}{{/if}}" data-notification={{this.alert.show}}>
```
Comment (micchickenburger) [inline packages/thriver-navigation/lib/templates/utility.html:28]: Can this be made more readable?

File: packages/thriver-core/lib/client/scroll.js (hunk 1 above)
Comment (eoghanTadhg) [reply packages/thriver-core/lib/client/scroll.js:97]: It seems it was causing error if the element did not exist. Will investigate why it does not exist.

File: packages/wcasa:work/lib/client/work.js [hunk 3]
```diff
@@ -70,6 +70,10 @@ var changeTabs = function (event) {
             });
         }, 250);
     }
+    //Scroll to top of work on change tabs
+    // TODO: Use smoothScroll
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:74]: New syntax for TODOs:

`// TODO(eoghantadhg): Use smoothscroll`

File: packages/wcasa:work/lib/client/work.js [hunk 4]
```diff
@@ -70,6 +70,10 @@ var changeTabs = function (event) {
             });
         }, 250);
     }
+    //Scroll to top of work on change tabs
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:73]: We now use airbnb's style guide.  Spaces are required after slashes:

`// Scroll to...`

File: packages/wcasa:work/lib/client/work.js [hunk 5]
```diff
@@ -192,6 +197,11 @@ Template.workNav.events({
         setTimeout(function () {
             document.body.classList.remove('workActive', 'workReading', 'workFadeOut');
         }, 200);
+
+        //Scroll to top of work on change tabs
+        // TODO: Use smoothScroll
+        offset = $('[id="what-we-do"]').offset().top + 228;
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:203]: Create a constant for value `228`, like:  `const OFFSET = 228;`

File: packages/wcasa:work/lib/client/work.js [hunk 6]
```diff
@@ -192,6 +197,11 @@ Template.workNav.events({
...
+        $('body').animate({ scrollTop: offset }, 750);
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:204]: Create a constant here, too.

**Review by micchickenburger — CHANGES_REQUESTED** [review]

**Comment by eoghanTadhg** [comment]
These changes have been absorbed by branch `wcasa-dev` via an alternative pull-request, `mobileUiDebug`.

**Pull request closed without merging**